### ISSUE TITLE
gcloud crashed (UnicodeDecodeError): 'ascii' codec can't decode byte 0xe2 in position 1479: ordinal not in range(128)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prototipando/google-cloud-sdk#385]
```
Issue running command [gcloud beta test android run].
Command line
blaze-bin/cloud/sdk/gcloud/gcloud beta test android run --app 
gs://tonyren/basicapp/app.apk --test gs://tonyren/basicapp/test.apk -b 
gs://tonyren --verbosity info

What steps will reproduce the problem?
Not sure, only seen this once before.

What is the expected output? What do you see instead?
Expecting the output to display information regarding test executions.

Please provide any additional information below.


Installation information:

Google Cloud SDK [0.9.75]

Platform: [Linux, x86_64]
Python Version: [2.7.6 (default, Jun 22 2015, 17:58:13)  [GCC 4.8.2]]
Site Packages: [Disabled]

Installation Root: [/usr/local/google/home/kaijren/google-cloud-sdk]
Installed Components:
  core: [2015.08.23]
  core-nix: [2015.06.02]
  gcloud: [2015.08.23]
  gsutil-nix: [4.12]
  gsutil: [4.13]
  bq: [2.0.18]
  bq-nix: [2.0.18]
System PATH: 
[/usr/local/google/home/kaijren/google-cloud-sdk/bin:/usr/local/google/home/kaij
ren/google-cloud-sdk/bin:/usr/lib/google-golang/bin:/usr/local/buildtools/java/j
dk/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin]
Cloud SDK on PATH: [True]

Installation Properties: 
[/usr/local/google/home/kaijren/google-cloud-sdk/properties]
User Config Directory: [/usr/local/google/home/kaijren/.config/gcloud]
User Properties: [/usr/local/google/home/kaijren/.config/gcloud/properties]
Current Workspace: [None]
Workspace Config Directory: [None]
Workspace Properties: [None]

Account: [kaijren@google.com]
Project: [focal-column-87918]

Current Properties:
  [test]
    results_base_url: [https://pantheon-test.corp.google.com]
  [core]
    project: [focal-column-87918]
    account: [kaijren@google.com]
    disable_usage_reporting: [True]
  [api_endpoint_overrides]
[output truncated]

```

Original issue reported on code.google.com by `kaij...@google.com` on 12 Nov 2015 at 1:56

**Comment by GoogleCodeExporter** [comment]
```
This was the output:

INFO: Test Service endpoint: [https://localhost:8443/]   
INFO: Tool Results endpoint: 
[https://www-googleapis-test.sandbox.google.com/toolresults/v1beta3/]

Have questions, feedback, or issues? Please let us know by using this Google 
Group:
  https://groups.google.com/forum/#!forum/google-cloud-test-lab-external

INFO: Raw results root path is: [gs://tonyren/2015-11-11_17:42:34.989821_iOAF/]
Uploading [gs://tonyren/basicapp/app.apk] to the Cloud Test Lab...
Uploading [gs://tonyren/basicapp/test.apk] to the Cloud Test Lab...
Raw results will be stored in your GCS bucket at 
[https://console.developers.google.com/storage/browser/tonyren/2015-11-11_17:42:
34.989821_iOAF/]

Test [matrix-gli3t7dz0vya] has been created in the Google Cloud.
Cloud Test Lab will execute your instrumentation test on 1 device(s).
Creating individual test executions...done.

Test results will be streamed to 
[https://pantheon-test.corp.google.com/project/focal-column-87918/testlab/mobile
/histories/bh.ca67df535f455a1c/executions/5560105090469448283].
Test is Pending
ERROR: gcloud crashed (UnicodeDecodeError): 'ascii' codec can't decode byte 
0xe2 in position 1479: ordinal not in range(128)

If you would like to report this issue, please run the following command:
  gcloud feedback
```

Original comment by `kaij...@google.com` on 12 Nov 2015 at 1:58

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed in 0.9.88, which is out now.
```

Original comment by `z...@google.com` on 24 Nov 2015 at 9:52
- Changed state: **Fixed**
